x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/action/TransportStopDataFrameTransformAction.java

```diff
@@ -52,17 +59,28 @@ public TransportStopDataFrameTransformAction(TransportService transportService,
     @Override
     protected void doExecute(Task task, StopDataFrameTransformAction.Request request,
             ActionListener<StopDataFrameTransformAction.Response> listener) {
-        // Need to verify that the config actually exists
-        dataFrameTransformsConfigManager.getTransformConfiguration(request.getId(), ActionListener.wrap(
-            config -> super.doExecute(task, request, listener),
-            listener::onFailure
+
+        dataFrameTransformsConfigManager.expandTransformIds(request.getId(), new PageParams(0, 10_000), ActionListener.wrap(
```

Comment:
TransportStopDataFrameTransformAction is not a master node action. The node where this request first arrives becomes the coordinating node. I don't think that is a problem correct me if I'm wrong.